### ISSUE TITLE
Logs are hard to read

### ISSUE THREAD

**Issue by emanuelen5** [makerspace/memberbooth#81]
There are several issues:

1. This is printed every minute when there is **no** activity
    ```python
    2020-10-06 00:25:55,084 INFO [/home/memberbooth/memberbooth/src/gui/states.py:40]: Event: gui_event_timeout_timer_expired, without data
    ```
2. This is printed every time the mouse/keyboard is used (can be several times a second while in use):
    ```python
    2020-10-05 23:12:47,762 INFO [/home/memberbooth/memberbooth/src/gui/design.py:87]: Timeout timer was reset
    ```
3. State transitions are hard to follow from the log (this shows when a member prints a label):
    ```python
    2020-10-05 23:12:47,972 INFO [/home/memberbooth/memberbooth/src/gui/states.py:40]: Event: gui_event_print_storage_label, with data
    ````

**Comment by InMyOrbit** [comment]
Things that we talked about fixing a year ago but never did... 

1 & 2 - Straight forward. 
3 - We should aim to have event and the transition in the logs, I agree.

**Comment by InMyOrbit** [comment]
Fixed 1&2 in ede7b631edafea2f2b1fefe39151b675bc7043ea.

Looking at 3 I think that is expected behavior, there is no state transition during the mentioned event, the state transition happens when the printing is finished. So the event mentioned is a signal from the gui to the application which then initiates the print and then causes a state transition based on the printing result.

**Comment by emanuelen5** [comment]
Closed in fdb496a
